### ISSUE TITLE
 I can only train youtube and davis with batch size=4

### ISSUE THREAD

**Issue by longmalongma** [hkchengrex/XMem#23]
Hi @hkchengrex Since my gpu memory is limited, I can only train youtube and davis with batch size=4, do you think this will affect the accuracy?

**Comment by hkchengrex** [comment]
The general advice is to decrease the learning rate and increase the total number of iterations to make up for the difference. You would have to try for yourself.
